src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -199,24 +200,89 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
 
   def getObjectMetadata(bucketName: String, objectKey: String, authToken: String)
                     (implicit actorRefFactory: ActorRefFactory, executionContext: ExecutionContext): Future[ObjectMetadata] = {
-    val metadataRequest = Get( getObjectResourceUrl(bucketName, objectKey) )
```

Comment:
Was the `GET` response body useful here?

Reply:
it certainly was useful, but we had to switch from the json api to the xml api due to pet SA/Google limitation. So I replaced the json `Get( getObjectResourceUrl ... )` with its equivalent in the xml api `Head( getXmlApiMetadataUrl ... )`